internal/pkg/template/templates/workloads/services/lb-web/cf.yml

```diff
@@ -170,6 +170,9 @@ Resources:
 {{- end}}
       Port: !Ref ContainerPort
       Protocol: HTTP
+{{- if .ProtocolVersion}}
+      ProtocolVersion: {{.ProtocolVersion}}
```

Comment:
is it possible that we accept any case for the value? Users should be able to write:
```
http:
  version: grpc
```

Reply:
OK. I will try.

Reply:
I tried to try this, but I am thinking about the timing to convert the case.

1. convert to uppercase when loading the manifest file
2. convert to uppercase when generating the cloud formation template

Depending on how I implement it, I need to change the validation logic you asked me to add `internal/pkg/manifest/lb_web_svc.go`.

Since CloudFormation allows only uppercase GRPC/HTTP1/HTTP2 for the Protocol Version, I think it simply makes the user enter uppercase.

Could you give some advice, please?

Reply:
No worries the code looks great!

The conversion between "manifest" input to "cluodformation" input happens in the `cloudformation/stack` pkg. For example:
https://github.com/aws/copilot-cli/blob/47a6e7e4b90a1977233895476100f5b363641876/internal/pkg/deploy/cloudformation/stack/lb_web_svc.go#L168

I've added an extra commit to the PR to make the change, thank you! 🎉